### PR TITLE
Add shorah to bioconda

### PR DESCRIPTION
**Pull request by SoapZA**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.

### CODE REVIEW

File: recipes/shorah/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+source:
+  fn: shorah-1.0.0.tar.bz2
+  url: https://github.com/cbg-ethz/shorah/releases/download/v1.0.0/shorah-1.0.0.tar.bz2
+  md5: f0d057d3947b533ef4308458de9bfa8d
+
+requirements:
+  build:
+   - gcc # [linux]
+   - llvm # [osx]
+   - python >=2.7,<3
```
Comment (bgruening) [inline recipes/shorah/meta.yaml]: This should be done by specifying `skip: True #[py3k]` in the build section.

Comment (SoapZA) [inline recipes/shorah/meta.yaml]: @bgruening have a look at the log, without python at build time the configure script bombs out trying to byte-compile the python modules.

Comment (bgruening) [inline recipes/shorah/meta.yaml]: Sorry for the confusion, I meant the `>=2.7,<3` can be dropped in favor of the other skip. Should I do the changes?

Comment (SoapZA) [inline recipes/shorah/meta.yaml]: @bgruening done, anything else you need before merging it?

**Comment by bgruening** [comment]
Perfect, thanks!!!

**Pull request merged**